output/mapping/F14_2014.csv

```diff
@@ -44,31 +44,31 @@ xpath,label-key,index,comment,guidance
 /COMPLEMENTARY_INFO/DATE_DISPATCH_NOTICE,date_dispatch,VI.5,,Map to `date`
 /COMPLEMENTARY_INFO/NOTICE_NUMBER_OJ,number_oj,,https://github.com/open-contracting-archive/trade/tree/master/draft_extensions/release_relatedNotice,[Reference a previous publication](../operations/#reference-a-previous-publication)
 /CHANGES,changes,VII,,"Add an `Amendment` object to the `tender.amendments` array, and set its `.id` (string) sequentially, across all F14 notices for this procedure. (The first notice sets it to '1', the second to '2', etc.)"
-/CHANGES/CHANGE,icar_text_corrected,VII.1.2,,See [Guidance for `/CHANGES/CHANGE`](#guidance-for-changes-change)
+/CHANGES/CHANGE,icar_text_corrected,VII.1.2,,"For each `CHANGE` element, add an `UnstructuredChange` object to the amendment's `.unstructuredChanges` array, and set its `.where`, `.oldValue` and `.newValue` as follows:"
 /CHANGES/CHANGE/WHERE,,,,""
-/CHANGES/CHANGE/WHERE/SECTION,section_no,,,""
+/CHANGES/CHANGE/WHERE/SECTION,section_no,,,Map to `.where.section`.
 /CHANGES/CHANGE/OLD_VALUE,icar_instead,,,""
 /CHANGES/CHANGE/NEW_VALUE,icar_read,,,""
-/CHANGES/CHANGE/WHERE/LOT_NO,lot_number,,,""
-/CHANGES/CHANGE/WHERE/LABEL,icar_text_corr_place,,,""
-/CHANGES/CHANGE/OLD_VALUE/NOTHING,,,,""
-/CHANGES/CHANGE/NEW_VALUE/NOTHING,,,,""
-/CHANGES/CHANGE/OLD_VALUE/TEXT,,,,""
-/CHANGES/CHANGE/NEW_VALUE/TEXT,,,,""
+/CHANGES/CHANGE/WHERE/LOT_NO,lot_number,,,Map to `.where.lot`.
+/CHANGES/CHANGE/WHERE/LABEL,icar_text_corr_place,,,Map to `.where.label`.
+/CHANGES/CHANGE/OLD_VALUE/NOTHING,,,,Set `.oldValue.text` to `null`.
+/CHANGES/CHANGE/NEW_VALUE/NOTHING,,,,Set `.newValue.text` to `null`.
+/CHANGES/CHANGE/OLD_VALUE/TEXT,,,,Map to `.oldValue.text`.
+/CHANGES/CHANGE/NEW_VALUE/TEXT,,,,Map to `.newValue.text`.
 /CHANGES/CHANGE/OLD_VALUE/CPV_MAIN,,,,""
 /CHANGES/CHANGE/NEW_VALUE/CPV_MAIN,,,,""
-/CHANGES/CHANGE/OLD_VALUE/CPV_MAIN/CPV_CODE,cpv_main,,,""
-/CHANGES/CHANGE/NEW_VALUE/CPV_MAIN/CPV_CODE,cpv_main,,,""
-/CHANGES/CHANGE/OLD_VALUE/CPV_MAIN/CPV_SUPPLEMENTARY_CODE,cpv_supplem,,,""
-/CHANGES/CHANGE/NEW_VALUE/CPV_MAIN/CPV_SUPPLEMENTARY_CODE,cpv_supplem,,,""
+/CHANGES/CHANGE/OLD_VALUE/CPV_MAIN/CPV_CODE,cpv_main,,,Map to `.oldValue.mainCpv`.
```

Comment:
We don't need both `mainCpv` and `additionalCpv`. (On the form, you have to choose one or the other, and the only difference is that additional CPVs are repeatable). This can just be a `classifications` array, that is mapped using `Classification` objects like elsewhere.

In any case, all the CPV guidance at least needs to have "take the CPV code in the `CODE` attribute", since these elements contain no text; they only have data in attributes.

Reply:
> On the form, you have to choose one or the other, and the only difference is that additional CPVs are repeatable

I don't see the additionnal CPV on the [PDF form](On the form, you have to choose one or the other, and the only difference is that additional CPVs are repeatable), however it's in the XML schema. Weird, isn't it?

Reply:
Ah now I get it. It's not main vs additionnal, it's main vs supplementary :dizzy_face:.

Reply:
My confusion partly came from the fact that in the EU profile, the supplementary codes are mapped to OCDS by adding them to the `additionalClassifications` array.

Reply:
> This can just be a classifications array

That means that if it's a `CPV_MAIN`, it will produce a single-entry `classifications` array, and it's as fine as a dedicated `Classification` property (`mainCPV`), but simpler, right?

Reply:
I think I nailed it with 5879f83. If not, let's call tomorrow :)

Reply:
@jpmckinney Could you please review my proposal? I'll start the extension but I'll need a stamped mapping to complete it.